Formula/crystal.rb

```diff
@@ -4,8 +4,8 @@ class Crystal < Formula
   revision 1
```

Comment:
This line causes the build to fail and should be removed :)

Reply:
Thanks!

Isn't it odd that `brew audit --strict` didn't complain about that?

Reply:
It should have if you ran it after having built and installed the new version.

Reply:
I might be doing something wrong @fxcoudert 

```
$ brew uninstall crystal
$ brew install --build-from-source Formula/crystal.rb
$ brew audit --strict Formula/crystal.rb 
$ brew audit --strict crystal
```

but non of the last two commands produce output on my side.

Anyway, I'm more interested in getting this merged if possible :-) Thanks!